parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -65,7 +65,11 @@ fi
 
 if [[ "${SHOULD_CONFIGURE_HTTP_PROXY}" == "true" ]]; then
     if [[ "${SHOULD_CONFIGURE_HTTP_PROXY_CA}" == "true" ]]; then
-        configureHTTPProxyCA || exit $ERR_UPDATE_CA_CERTS
+        if [[ $OS == $MARINER_OS_NAME ]]; then
```

Comment:
can we move this inside the func and avoid pivoting on the positional params? feels like recipe for error in ordering or similar

Reply:
yeees, was actually thinking the same :)